static/js/zamboni/buttons.js

```diff
@@ -403,10 +403,12 @@ var installButton = function() {
         var opts = search ? {addPopup: false, addWarning: false} : {};
         versionsAndPlatforms(opts);
     } else if (z.app == 'firefox') {
+        $('#downloadAnyway').attr('href',escape_($button.filter(':visible').attr('href')));
```

Comment:
Did you use tabs for the indentation or something? It appears correctly indented in the [full view](https://github.com/FrancescoSTL/olympia/blob/bug-1187097-download-anyway/static/js/zamboni/buttons.js) but not here